hikari/api/rest.py

```diff
@@ -2329,7 +2329,7 @@ async def fetch_ban(
         self,
         guild: snowflakes.SnowflakeishOr[guilds.PartialGuild],
         user: snowflakes.SnowflakeishOr[users.PartialUser],
-    ) -> guilds.GuildMemberBan:
+    ) -> typing.Optional[guilds.GuildMemberBan]:
```

Comment:
You'd need to update your docs in the other PR then right?

Reply:
Yea, at the time I did this commit I couldnt rebase to master. Its the reason I left it as a draft 😄